### PR TITLE
set QEMU_CPU so uname will report correct information

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

File: debian/entry.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+function remove_buildtime_env_var()
+{
+	unset QEMU_CPU
```
Comment (petrosagg) [inline debian/entry.sh]: why unset the variable? We want it to persist

Comment (nghiant2710) [inline debian/entry.sh]: @petrosagg, I think on device QEMU will no longer emulate and this command will report correct information without this env var

Comment (petrosagg) [inline debian/entry.sh]: you're right. I thought this is build-time

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**